deploy/zookeeper/quick-start-persistent-volume/zookeeper-1-node.yaml

```diff
@@ -139,7 +142,9 @@ spec:
                 echo 'snapCount=3000000'
                 echo 'leaderServes=yes'
                 echo 'standaloneEnabled=true'
```

Comment:
standaloneEnabled=true, should be false. I also do not see the server.1 change here.

Reply:
fixed on https://github.com/Altinity/acm/commit/9455cf6e87a95de7c0ff36c221d3fe88988c1f79